tests/packages/media-sound/bitwig-studio-1.3.16.conf

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cat >/etc/portage/package.use/bitwig-studio <<EOF
```

Comment:
I must ask, is THIS really a blocker ? 
I mean we are talking about code style here, on a level that is quite detailed.